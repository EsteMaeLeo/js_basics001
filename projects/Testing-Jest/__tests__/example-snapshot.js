const client = {
  name: "William Walkings",
  budget: 5000,
  type: "Premium",
};

describe("Testing client", () => {
  test("Client is William", () => {
    expect(client).toMatchSnapshot();
  });
});
