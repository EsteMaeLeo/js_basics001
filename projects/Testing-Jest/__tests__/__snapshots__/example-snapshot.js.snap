// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing client Client is William 1`] = `
{
  "budget": 5000,
  "name": "William Walkings",
  "type": "Premium",
}
`;
